test/extended/testdata/builds/subscription-content/buildconfig-subscription-content-rhel9.yaml

```
@@ -32,7 +32,7 @@ spec:
     dockerfile: |
       FROM registry.access.redhat.com/ubi9/ubi:latest
       RUN rm -rf /etc/rhsm-host
-      RUN yum --enablerepo=codeready-builder-for-rhel-9-x86_64-rpms install \
+      RUN yum --enablerepo=codeready-builder-for-rhel-9-$arch-rpms install \
```

Use output from `arch` command:

```suggestion
      RUN yum --enablerepo=codeready-builder-for-rhel-9-$(arch)-rpms install \
```